homeassistant/components/brottsplatskartan/__init__.py

```diff
@@ -1 +1,111 @@
-"""The brottsplatskartan component."""
+"""Component for Brottsplatskartan information."""
+import uuid
+
+import voluptuous as vol
+
+from homeassistant.const import (ATTR_ATTRIBUTION, CONF_LATITUDE,
+                                 CONF_LONGITUDE, CONF_MONITORED_CONDITIONS,
+                                 CONF_NAME)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.dispatcher import dispatcher_send
+from homeassistant.helpers.event import track_time_interval
+from homeassistant.util import slugify
+
+from .const import (_LOGGER, ATTR_INCIDENTS, CONF_AREAS, CONF_SENSOR,
+                    DEFAULT_NAME, DEFAULT_SCAN_INTERVAL, DOMAIN, SENSOR_TYPES,
+                    SIGNAL_UPDATE_BPK)
+
+AREAS = [
+    "Blekinge län", "Dalarnas län", "Gotlands län", "Gävleborgs län",
+    "Hallands län", "Jämtlands län", "Jönköpings län", "Kalmar län",
+    "Kronobergs län", "Norrbottens län", "Skåne län", "Stockholms län",
+    "Södermanlands län", "Uppsala län", "Värmlands län", "Västerbottens län",
+    "Västernorrlands län", "Västmanlands län", "Västra Götalands län",
+    "Örebro län", "Östergötlands län"
+]
+
+SENSOR_SCHEMA = vol.Schema({
+    vol.Optional(CONF_MONITORED_CONDITIONS, default=[]):
```

Comment:
We should probably remove this config option to follow https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md.